index.html

```diff
@@ -1,235 +1,213 @@
-                            LET'S CONNECT
-                          </p>
+            <!-- About myself -->
+            <footer class="GridCard51a">
+                <section class="GridWrapper51a">
+                    <section class="last" id="About">
+                        <div>
+                            <h2>About <br> Myself</h2>
+                            <p>Hello, I'm a software developer! I can help you build a product, feature or
+                                website. Look through some of my work and experience! if you like what you see
+                                and have a project you need coded, dont hesitate to contact me.
+                            </p>
+                            <h3>LETS CONNECT</h3>
+                            <ul class="social-media48">
+                                <li> <a href="#"><img src="./images/github.jpg" alt="github"></a> </li>
+                                <li> <a href="#"><img src="./images/medium.jpg" alt="Linkedin"></a> </li>
+                                <li> <a href="#"><img src="./images/twitter.jpg" alt="medium"></a> </li>
+                                <li> <a href="#"><img src="./images/Vector.jpg" alt="twitter"></a> </li>
+                            </ul>
+
+                            <div class="click52">
+                                <button>Get my resume</button>
+                            </div>
+                        </div>
+                        <ul class="oye">
+                            <li class="names22">
+                                <h4>Languages</h4><img src="./images/chevron-down-solid.svg" alt="">
+                            </li>
+
+                            <li class="agbe">
+                                <img src="./images/Ellipse-1.jpg" alt="work">
+                                <h5>Javascript</h5>
+                            </li>
+
+                            <li class="agbe">
+                                <img src="./images/Ellipse-12.jpg" alt="work">
+                                <h5>HTML</h5>
+                            </li>
+                            <li class="agbe">
+                                <img src="./images/Ellipse-9.jpg" alt="work">
+                                <h5>CSS</h5>
+                            </li>
+
+                            <li class="names77">
+                                <h4>Frameworks</h4><img src="./images/chevron-right-solid.svg" alt="works">
+                            </li>
+                            <li class="names77">
+                                <h4>Skills</h4><img src="./images/chevron-right-solid.svg" alt="works">
+                            </li>
+                        </ul>
+                    </section>
+                </section>
+            </footer>
+            <footer class="GridCardlst">
+                <section class="GridWrapperlst" id="contact">
+                    <h2>Contact me</h2>
```

Comment:
- Optional
Your contact me section is also not in accordance to the Figma template of your choosing. Kindly fix it.
![image](https://user-images.githubusercontent.com/33268927/175068649-2ce28ed5-a0db-4f1f-ae4b-a86806a25227.png)